modules/red-hat-marketplace-features.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * applications/red-hat-marketplace.adoc
+
+[id="red-hat-marketplace-features_{context}"]
+= Red Hat Marketplace features
+
+Cluster administrators can use link:https://marketplace.redhat.com/en-us/documentation/getting-started[Red Hat Marketplace] to manage software on {product-title}, give developers self-service access to deploy application instances, and correlate application usage against a quota.
+
+[id="marketplace-clusters_{context}"]
+== Connect {product-title} clusters to the Marketplace
+
+You can install a common set of applications on {product-title} clusters that you connect to the Marketplace. You can also use the Marketplace to track cluster usage against your subscriptions or quota. Users that you add by using the Marketplace have their product usage tracked and billed to your organization.
+
+During the link:https://marketplace.redhat.com/en-us/documentation/clusters[cluster connection process],
+a Marketplace Operator is installed that updates the image registry secret, manages the catalog, and reports application usage.
+
+[id="marketplace-install-applications_{context}"]
+== Install applications
+
+Cluster administrators can link:https://marketplace.redhat.com/en-us/documentation/operators[install Marketplace applications] from within OperatorHub in {product-title}, or at the link:https://marketplace.redhat.com[Marketplace web application].
+
+Access installed applications from the web console under *Installed Operators*.
+
+[id="marketplace-deploy_{context}"]
+== Deploy applications from different perspectives
+
+Deploy Marketplace applications from the web console's Administrator and Developer perspectives.
+
+[discrete]
+=== The Developer perspective
+
+Access newly installed capabilities from the Developer perspective’s catalog.
+
+For example, after a database Operator is installed, a developer can create an instance from the catalog within their Project. Database usage is aggregated and reported to the cluster administrator.
+
+This perspective does not include Operator installation and application usage tracking.
+
+[discrete]
+=== The Administrator perspective
+
+The Administrator perspective includes information about Operator installation and application usage.
+
+{product-title} administrators can launch application instances by browsing custom resource definitions (CRDs) within the *Installed Operators* list.
```

Comment:
Should this just say "Cluster administrators" like we say elsewhere in this module?